docs/admin-panel/hierarchy/ERC721/contract.md

```diff
@@ -15,7 +15,13 @@ ERC721 contracts are being deployed using [Contract Manager](/docs/admin-panel/m
 This information is passed directly to blockchain
 
 - **template** - Smart contract template with its unique functions
-    - **SIMPLE** - Simple template includes all basic functions like _mint_, _burn_ and _transfer_
+    - **SIMPLE** - Simple template includes all basic functions like ***mint***, ***burn*** and ***transfer***. Consist of several contracts:
```

Comment:
and several extensions

Reply:
Already different structure. I don't mention about any features in contract page.
I will add this change in features page